index.html

```diff
@@ -1,11 +1,151 @@
+          <h3 class="item__title">СУЧАСНІ ТЕХНОЛОГІЇ</h3>
+
+          <p class="item__description">
+            Значимість цих проблем настільки очевидна, що реалізація планових
+            завдань.
+          </p>
+        </article>
+      </section>
+
+      <section class="main__services services">
+        <h2 class="services__title title">Чим ми займаємося</h2>
+
+        <div class="services__container">
+          <img
+            class="services__image"
+            src="./images/services--1.jpg"
+            alt="First image in services"
+            width="370px"
+          />
+        </div>
+        <div class="services__container">
+          <img
+            class="services__image"
+            src="./images/services--2.jpg"
+            alt="Second image in services"
+            width="370px"
+          />
+        </div>
+        <div class="services__container">
+          <img
+            class="services__image"
+            src="./images/services--3.jpg"
+            alt="Third image in services"
+            width="370px"
+          />
+        </div>
+      </section>
+
+      <section class="main__team team">
+        <h2 class="team__title title">Наша команда</h2>
+        <ul class="team__list">
+          <li class="team__card card">
+            <img
+              class="card__image"
+              src="./images/team--1.jpg"
+              alt="First image in cards"
+              width="270px"
+            />
+            <p class="card__name">Ігор Дем'яненко</p>
+            <p class="card__skill">Product Designer</p>
+          </li>
+          <li class="team__card card">
+            <img
+              class="card__image"
+              src="./images/team--2.jpg"
+              alt="Second image in cards"
+              width="270px"
+            />
+            <p class="card__name">Ольга Рєпіна</p>
+            <p class="card__skill">Frontend Developer</p>
+          </li>
+          <li class="team__card card">
+            <img
+              class="card__image"
+              src="./images/team--3.jpg"
+              alt="Third image in cards"
+              width="270px"
+            />
+            <p class="card__name">Микола Тарасов</p>
+            <p class="card__skill">Marketing</p>
+          </li>
+          <li class="team__card card">
+            <img
+              class="card__image"
+              src="./images/team--4.jpg"
+              alt="Fourth image in cards"
+              width="270px"
+            />
+            <p class="card__name">Михайло Єрмаков</p>
+            <p class="card__skill">UI Designer</p>
+          </li>
+        </ul>
+      </section>
+    </main>
+    <footer class="footer">
+      <a class="footer__logo logo" href="#">WebStudio</a>
+
+      <address class="footer__address">м. Київ, пр-т Лесі Українки, 26</address>
+
+      <a class="footer__email email" href="#">info@devstudio.com</a>
+
+      <p class="footer__number namber">+38 096 111 11 11</p>
```

Comment:
this should be a separate block - `contacts`, modify the class names accordingly